dev/ContentDialog/ContentDialog.vcxitems

```diff
@@ -13,7 +13,7 @@
   <ItemGroup>
     <ProjectCapability Include="SourceItemsFromImports" />
   </ItemGroup>
-  <ItemGroup Condition="$(BuildLeanMuxForTheStoreApp) != 'true'">
+  <ItemGroup>
```

Comment:
This seems unrelated to the subject of this PR. Is its inclusion intended?

Reply:
I think this was a part of a merge from master, specifically resolving conflicts with merging in changes to the build system.

Reply:
This is ok.